json_schema-vllm-direct_6.py

```diff
@@ -0,0 +1,21 @@
+from pydantic import BaseModel
+import json
+from openai import OpenAI
+
+class CompletionResponse(BaseModel):
+    completion_message: str
+
+client = OpenAI(base_url="http://localhost:8000/v1", api_key="fake")
+
+completion = client.chat.completions.create(
+    model="meta-llama/Llama-3.2-3B-Instruct",
+    messages=[
+        {
+            "role": "user",
+            "content": "Write me a print statement that prints 'Hello, World!' in python",
+        },
+    ],
+    response_format={"type": "json_object"}
+)
```

Comment:
Hangs and doesn't complete 🤔 

<img width="883" alt="image" src="https://github.com/user-attachments/assets/f3be4390-1626-447c-879e-4bd1ea49e2ee">